src/components/shared/DownloadButton.tsx

```diff
@@ -42,22 +43,40 @@ export default function DownloadButton(props: DownloadButtonProps) {
     }
     
     return (
-        <Button key={buttonLabelKey} size={buttonSize} className="download-data-btn">
+        iconName === "" ?       
```

Comment:
Since this needs to be done on a time crunch I won't let this block merge. But I don't think this is an extensible or clear design - nobody will understand that iconName == "" translates to the airtableLogo.

Moving forward I think we should let the user specify some jsx that would be rendered as the "iconComponent", or EITHER an iconName or iconSource.

Reply:
@annoyingpr can we make this a flup ticket